config/initializers/devise.rb

```diff
@@ -0,0 +1,311 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'cff8092770af8c01d404cd136e137e6a364d7f2d7fb4f2620e39407387f696bff8bbde59fb1f8fe3b23fcf89d11292f744cf4c1346bfd5ce4cc3febc49c421e3'
+
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+
+  # Configure the class responsible to send e-mails.
+  # config.mailer = 'Devise::Mailer'
```

Comment:
- [ ] When I try to sign up I didn't get any confirmation email So I can't log in to the app, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/197374446-9580a28d-6272-4bc9-9438-93312484bef0.png)

Reply:
@Omar-Muhamad Thank for your review. Please check your terminal to find the confirmation link

Reply:
It's supposed to be sent to the registered email or you can use a gem to make the content of the confirmation email opened on a new tab like mailboxer gem.

Reply:
@Omar-Muhamad, thank you again for your suggestions. I'll implement it.